src/java/fr/paris/lutece/plugins/search/solr/service/SolrSearchAppConfService.java

```diff
@@ -80,6 +81,10 @@ public static SolrSearchAppConf loadConfiguration( String code )
             {
                 conf.setFilterQuery( referenceItemName );
             }
+            else if ( referenceItemCode.matches( DSKEY_FQ_REGULAR_EXPR ) )
+            {
+                conf.addFilterQuery( referenceItemName );
```

Comment:
Need to update SaveConfiguration as well even though it's not called yet.
Adding tests would be an even better bonus :)